test/unit/db/repos/accounts.js

```diff
@@ -48,871 +50,1404 @@ describe('db', () => {
+					.then(() => {
+						return db.one(
+							'SELECT COUNT(*)::int AS count FROM mem_accounts2multisignatures'
+						);
+					})
+					.then(row => {
+						expect(row.count).to.equal(0);
+					});
+			});
+
+			it('should empty the table "mem_accounts2u_multisignatures"', () => {
+				return db.accounts
+					.resetMemTables()
+					.then(() => {
+						return db.one(
+							'SELECT COUNT(*)::int AS count FROM mem_accounts2u_multisignatures'
+						);
+					})
+					.then(row => {
+						expect(row.count).to.equal(0);
+					});
+			});
+		});
+
+		describe('list()', () => {
+			before(() => {
+				return createAccount().then(account => {
+					validAccount = account;
+				});
+			});
+
+			it('should return data without any error', () => {
+				return db.accounts.list();
+			});
+
+			describe('fields', () => {
+				it('should return all table fields if no field is specified', () => {
+					return db.accounts.list().then(data => {
+						const columnNames = _.map(db.accounts.cs.select.columns, column => {
+							return column.prop || column.name;
 						});
 
-						it('should return "balance" as "bigint"', () => {
-							return db.accounts
-								.list({}, ['balance'], { limit: 1 })
-								.then(data => {
-									expect(data[0].balance).to.be.a('string');
-								});
+						expect(data[0]).to.have.all.keys(columnNames);
+					});
+				});
+
+				it('should return only "address" if fields specify ["address"]', () => {
+					return db.accounts.list({}, ['address']).then(data => {
+						data.forEach(account => {
+							expect(account).to.have.all.keys(['address']);
 						});
+					});
+				});
 
-						it('should return "u_balance" as "bigint"', () => {
-							return db.accounts
-								.list({}, ['u_balance'], { limit: 1 })
-								.then(data => {
-									expect(data[0].u_balance).to.be.a('string');
-								});
+				it('should return only "address" and "isDelegate" if fields specify ["address", "isDelegate"]', () => {
+					return db.accounts.list({}, ['address', 'isDelegate']).then(data => {
+						data.forEach(account => {
+							expect(account).to.have.all.keys(['isDelegate', 'address']);
 						});
+					});
+				});
 
-						it('should return "rate" as "bigint"', () => {
-							return db.accounts.list({}, ['rate'], { limit: 1 }).then(data => {
-								expect(data[0].rate).to.be.a('string');
+				it('should skip any unknown field specified ["address", "unKnownField"]', () => {
+					return db.accounts
+						.list({}, ['address', 'unKnownField'])
+						.then(data => {
+							data.forEach(account => {
+								expect(account).to.have.all.keys(['address']);
 							});
 						});
+				});
 
-						it('should return "fees" as "bigint"', () => {
-							return db.accounts.list({}, ['fees'], { limit: 1 }).then(data => {
-								expect(data[0].fees).to.be.a('string');
-							});
-						});
+				describe('property names', () => {
+					it('should return "producedBlocks" column if "producedBlocks" is asked', () => {
```

Comment:
`is requested`

Reply:
These tests are removed in PR https://github.com/LiskHQ/lisk/pull/1572 due to renaming the "producedBlocks" column.